tests/test_aaa_authorization_service.rb

```diff
@@ -22,16 +22,17 @@ class TestAaaAuthorizationService < CiscoTestCase
 
   def setup
     super
-    cleanup_aaa if @@pre_clean_needed
-    @@pre_clean_needed = false # rubocop:disable Style/ClassVars
-    feature_tacacs
+    if @@pre_clean_needed
+      cleanup_aaa
+      config('feature tacacs+')
+      @@pre_clean_needed = false # rubocop:disable Style/ClassVars
+    end
     preconfig_tacacs_server_access(tacacs_groups[0])
     config_tacacs_servers(tacacs_groups[1..3])
   end
 
   def teardown
     cleanup_aaa
-    feature_tacacs(false)
```

Comment:
Main rational is that configuring the feature to be on/off manually doesn't improve the test and wastes time.  Would rather set it once and have quicker tests since the test quality isn't affected by this change.